pkg/component/worker/kubelet.go

```diff
@@ -267,3 +302,54 @@ func getNodeName(ctx context.Context) (string, error) {
 	}
 	return string(h), nil
 }
+
+func parseTaint(st string) (corev1.Taint, error) {
```

Comment:
Was there no easy-to-use helpers for this functionality in kube modules? 

Would be nice to see some unit tests for this

Reply:
It's actually a cut'n'paste from the kube module :) 
Unfortunately, it's not a public method.